_posts/2024-04-15-trino-fest-2024-approaches.md

```diff
@@ -0,0 +1,124 @@
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/announcement-banner.png
+show_pagenav: true
+---
+
+Trino Fest is drawing ever closer, and Commander Bun Bun has been hard at work
+behind the scenes arranging the schedule and making sure that Trino's trip to
+Boston is going to be a great one. In case you missed it,
+[we announced Trino Fest]({{site.baseurl}}{% post_url 2024-02-20-announcing-trino-fest-2024 %})
+a couple months ago, and if you have missed it, make sure to go register to
+attend! All our speakers will be in person in downtown Boston on the 13th of
+June, with plenty of opportunities for networking and a 2-hour happy hour at the
+end of the day. But if you can't make the trip up to enjoy the lovely New
+England summer, we'll also be livestreaming the event, and you can register to
+join us virtually.
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-orange" href="http://www.starburst.io/info/trino-fest-2024?utm_medium=trino&utm_source=website&utm_campaign=Global-FY25-Q2-EV-Trino-Fest-2024&utm_content=Blog-2">
+        Register to attend!
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+Still on the fence, though? Read on for a sneak peek at our speaker lineup and
+previews of their talks. Companies being represented include names like Apple 
+and Amazon, so you know it's going to be good.
+
+<!--more-->
+
+## Data lakes
+
+It's no secret that using Trino as part of your data lake has become one of its
+major use cases in the past few years. We're excited to say that at Trino Fest,
+we'll have representation for each of the modern big three: Iceberg, Delta Lake,
+and Hudi.
+
+### Iceberg
+
+Iceberg will be covered twice: Amogh Jahagirdar from Tabular will be diving into
+the world of Iceberg views and how they can be leveraged to coordinate across
+different query languages and dialects; Amit Gilad from Cloudinary will be
+covering the story of migrating out of Snowflake to the wonderful world of open
+table formats and Iceberg.
+
+### Delta Lake
+
+Marius Grama, a Trino maintainer at Starburst, will be going into detail on the
+history, development, and improvements to the Delta Lake connector. With time
+travel for the Delta Lake connector landed in Trino 445, it's one of the most
+exciting areas for development in open source Trino, and there's some
+interesting technical stories that Marius is excited to share with the
+community.
+
+### Hudi
+
+Rounding out data lakes, Ethan Guo from Onehouse will be diving into Trino's
+Hudi connector, giving an update on what's been landed lately to improve
+performance and functionality, and he'll also be giving a preview of what's
+coming soon. The features are flying in, and if you're a current or prospective
+user of Hudi with Trino, you won't want to miss out.
+
+### Amazon S3
+
+And for all your data lake needs, Dai Ozaki from Amazon will be discussing best
+practices for running Trino on S3 data lakes - no matter which storage system
```

Comment:
```suggestion
practices for running Trino with S3 data lakes - no matter which storage system
```

Reply:
or "S3-backed data lakes"

Reply:
Also no matter which table format .. or whatever .. S3 is the storage system ...